get-started/the-stack.md

```diff
@@ -7,97 +7,147 @@ mapped_pages:
 
-Use {{kib}} to query and visualize the data that’s stored in {{es}}. Or, use the {{es}} clients to access data in {{es}} directly from common programming languages.
+### {{agent}} and {{integrations}}[stack-components-agent]
 
-#### {{kib}} [stack-components-kibana]
+{{agent}} is a single, unified way to add monitoring for logs, metrics, and other types of data to a host.
+It can also protect hosts from security threats, query data from operating systems, and forward data from remote services or hardware.
+Each agent has a single policy to which you can add [integrations](integration-docs://reference/index.md) for new data sources, security protections, and more.
+You can also use [{{agent}} processors](/reference/fleet/agent-processors.md) to sanitize or enrich your data.
 
-{{kib}} is the tool to harness your {{es}} data and to manage the {{stack}}. Use it to analyze and visualize the data that’s stored in {{es}}. {{kib}} is also the home for the Search, Observability and Security solutions. [Learn more about {{kib}}](/explore-analyze/index.md).
+{{fleet}} enables you to centrally manage {{agents}} and their policies.
+Use {{fleet}} to monitor the state of all your {{agents}}, manage agent policies, and upgrade {{agent}} binaries or integrations.
 
-#### {{es}} clients [stack-components-elasticsearch-clients]
+[Learn more about {{agent}}](/reference/fleet/index.md).
 
-The clients provide a convenient mechanism to manage API requests and responses to and from {{es}} from popular languages such as Java, Ruby, Go, Python, and others. Both official and community contributed clients are available. [Learn more about the {{es}} clients](/reference/elasticsearch-clients/index.md).
+### {{product.apm}} [stack-components-apm]
 
-## Version compatibility
-```{applies_to}
-deployment:
-  self:
-```
+{{product.apm}} is an application performance monitoring system.
+It allows you to monitor software services and applications in real-time by collecting detailed performance information on response time for incoming requests, database queries, calls to caches, external HTTP requests, and more.
+This makes it easy to pinpoint and fix performance problems quickly.
 
-:::{include} /deploy-manage/deploy/_snippets/stack-version-compatibility.md
-:::
+[Learn more about {{product.apm}}](/solutions/observability/apm/index.md).
+
+### OpenTelemetry Collector [stack-components-otel]
+
+[OpenTelemetry](https://opentelemetry.io/docs)(OTel) is a vendor-neutral observability framework for collecting, processing, and exporting telemetry data. Elastic is a member of the Cloud Native Computing Foundation (CNCF) and active contributor to the OpenTelemetry project.
 
-## Installation order
-```{applies_to}
-deployment:
-  self:
-```
+In addition to supporting upstream OTel development, Elastic provides Elastic Distributions of OpenTelemetry (EDOT), specifically designed to work with {{product.observability}}.
```

Comment:
I don't know if this is particularly related to our open source contributions here. try to center the value prop